src/components/Layout/index.tsx

```diff
@@ -8,14 +8,13 @@ export type LayoutProps = {
 
 const Wrapper = styled.div`
   max-width: 1440px;
-  height: 100vh;
+  height: 100%;
   display: grid;
   grid-template-columns: 1fr;
   grid-template-rows: auto 1fr auto;
-  margin: 0 auto;
-  padding: 72px 0px;
+  margin: 0px auto;
   @media (max-width: 1440px){
```

Comment:
e se isso aqui mudar de 1440px pra outro valor? Por exemplo: Se o container mudar, tu vai sar mudando tudo manualmente? Não é melhor ter isso dentro do tema por exemplo?

Reply:
What you think about [this](https://github.com/joaooliveiradev/conduit-frontend/pull/39/commits/f54337cdfc5c74d256ae4036f4843d0f7b9f5f11)